### PR TITLE
[FTheoryTools]Small QOL changes 2

### PR DESCRIPTION
**Pull request by emikelsons**
- Updated the FTheoryTools introduction page
- Improved parsing of parametric models(now the language in the model_description isn't as restricted, but everything inside any brackets will still be evaluated)
- weierstrass_model(h::HypersurfaceModel) now also constructs the appropriate model, if it has been specified in the database
- Renamed related_models to birational_models and associated_models now only refer to models sharing the same paper
- Added a function model_index that returns the database index of a literature model for future use

### CODE REVIEW

File: experimental/FTheoryTools/docs/src/introduction.md [hunk 1]
```diff
@@ -50,12 +50,17 @@ We anticipate the following workflow:
...
 
-We are also actively expanding our database of supported literature models. At this point, our database includes the models discussed in [LS13](@cite). Consequently, we are already covering infinite families of models (e.g., with $SU(k)$ gauge group, where $k \geq 1`).
-Still, the total number of papers in our database is at this point limited to about $5$. This is to be extended a lot.
+We are also actively expanding our database of supported literature models. At this point, amongst others, our database includes models from the following papers:
+- The Tate Form on Steroids: Resolution and Higher Codimension Fibers [LS13](@cite),
+- F-Theory on all Toric Hypersurface Fibrations and its Higgs Branches [KM-POPR15](@cite),
+- Quadrillion F-Theory Compactifications with the Exact Chiral Spectrum of the Standard Model [CHLLT19](@cite).
+
+Consequently, we are already covering infinite families of models (e.g., with $SU(k)$ gauge group, where $k \geq 1$).
+Still, the total number of papers in our database is at this point limited to about $45$. This is to be extended a lot.
```
Comment (HereAround) [inline experimental/FTheoryTools/docs/src/introduction.md]: I think this number is not correct (although I did not check it). But I thought we have about 45 different models (including such models that give us an infinite family, say SU(n)), but that the number of papers from where those models originate is more like 10.

File: experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl [hunk 2]
```diff
@@ -365,12 +365,12 @@ end
 
 
 @doc raw"""
-    associated_literature_models(m::AbstractFTheoryModel)
+    birational_literature_models(m::AbstractFTheoryModel)
 
-Return a list of the unique identifiers any `associated_literature_models` of
+Return a list of the unique identifiers any `birational_literature_models` of
```
Comment (HereAround) [inline experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl]: Maybe I am reading it wrong, but the sentence does not make sense to me. Is it missing a word or two?

File: experimental/FTheoryTools/docs/src/introduction.md (hunk 1 above)
Comment (emikelsons) [reply experimental/FTheoryTools/docs/src/introduction.md]: You're right I misread this originally

File: experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl [hunk 3]
```diff
@@ -379,14 +379,14 @@ Assuming that the first row of the given grading is the grading under Kbar
...
+julia> birational_literature_models(m)
 1-element Vector{String}:
  "1507_05954-1"
 ```
 """
-function associated_literature_models(m::AbstractFTheoryModel)
-  @req has_associated_literature_models(m) "No associated models known for this model"
-  return get_attribute(m, :associated_literature_models)
+function birational_literature_models(m::AbstractFTheoryModel)
+  @req has_birational_literature_models(m) "No birational models known for this model"
```
Comment (HereAround) [inline experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl]: Maybe "No birationally equivalent models known for this model"?

File: experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl (hunk 2 above)
Comment (HereAround) [reply experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl]: Maybe it should be "... OF any..."?

File: experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl [hunk 4]
```diff
@@ -805,9 +805,30 @@ julia> related_literature_models(m)
...
-  @req has_related_literature_models(m) "No related models known for this model"
-  return get_attribute(m, :related_literature_models)
+function associated_literature_models(m::AbstractFTheoryModel)
+  @req has_associated_literature_models(m) "No associated models known for this model"
+  return get_attribute(m, :associated_literature_models)
+end
+
+
+@doc raw"""
+    model_index(m::AbstractFTheoryModel)
```
Comment (HereAround) [inline experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl:815]: I like this! Thank you.

File: experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl (hunk 3 above)
Comment (emikelsons) [reply experimental/FTheoryTools/src/AbstractFTheoryModels/attributes.jl]: That does sound better

**Comment by HereAround** [comment]
I just checked the introduction page https://docs.oscar-system.org/dev/Experimental/FTheoryTools/introduction/. A couple more suggestions:
* Mikelis, please add yourself to the list of people to reach out to. (Currently only Andrew and I are listed.)
* Maybe we can also remove the line break before "Alternatively, you can [raise an issue on github](https://www.oscar-system.org/community/#how-to-report-issues)."
* Could we please extend the funding information (very bottom of the page): "Martin Bies and Mikelis Mikelsons appreciate support by the [TU-Nachwuchsring](https://rptu.de/en/tu-nachwuchsring-network-for-young-scientists-support/home-page)".

Related question for @apturner: Is your funding information as stated on the introduction page still up-to-date?

**Comment by HereAround** [comment]
I left a few nitpicks above. Generally speaking though, I really like those changes. Thank you for working on this @emikelsons !

**Comment by apturner** [comment]
> Related question for @apturner: Is your funding information as stated on the introduction page still up-to-date?

This is still accurate until the end of August, at which point it will change. For now, we can leave it as it is.

File: experimental/FTheoryTools/docs/src/introduction.md [hunk 5]
```diff
@@ -76,13 +81,14 @@ Future extensions include, but are not necessarily limited to, the following:
 
 Please direct questions about this part of OSCAR to the following people:
 * [Martin Bies](https://martinbies.github.io/),
-* [Andrew Turner](https://apturner.net/).
+* [Andrew Turner](https://apturner.net/),
+* [Mikelis Emils Mikelsons](https://github.com/emikelsons).
```
Comment (HereAround) [inline experimental/FTheoryTools/docs/src/introduction.md]: I would suggest alphabetic order by last name. Unless I am mistaken, this is the accepted standard OSCAR-wide.

Comment (HereAround) [reply experimental/FTheoryTools/docs/src/introduction.md]: So Bies, Mikelsons, Turner.

File: experimental/FTheoryTools/docs/src/introduction.md [hunk 6]
```diff
@@ -76,13 +81,14 @@ Future extensions include, but are not necessarily limited to, the following:
...
-You can ask questions in the [OSCAR Slack](https://www.oscar-system.org/community/#slack).
 
+You can ask questions in the [OSCAR Slack](https://www.oscar-system.org/community/#slack).
 Alternatively, you can [raise an issue on github](https://www.oscar-system.org/community/#how-to-report-issues).
 
 
 ## Acknowledgements
 
-We appreciate insightful discussions with [Mirjam Cvetič](https://live-sas-physics.pantheon.sas.upenn.edu/people/standing-faculty/mirjam-cvetic). The work of Andrew Turner is supported by DOE (HEP) Award DE-SC001352.
+We appreciate insightful discussions with [Mirjam Cvetič](https://live-sas-physics.pantheon.sas.upenn.edu/people/standing-faculty/mirjam-cvetic). Martin Bies and Mikelis Mikelsons appreciate support by the TU-Nachwuchsring. The work of Andrew Turner is supported by DOE (HEP) Award DE-SC001352.
```
Comment (HereAround) [inline experimental/FTheoryTools/docs/src/introduction.md:93]: Very good. Thank you!

**Review by HereAround — APPROVED** [review]
Thank you for working on this @emikelsons . Looks good to me!

**Pull request merged**